src/libraries/System.Net.Quic/src/System/Net/Quic/QuicConnection.cs

```diff
@@ -81,7 +81,7 @@ public static async ValueTask<QuicConnection> ConnectAsync(QuicClientConnectionO
     /// <summary>
     /// Handle to MsQuic connection object.
     /// </summary>
-    private MsQuicContextSafeHandle _handle;
+    private readonly MsQuicContextSafeHandle _handle;
```

Comment:
We should also made `readonly` the same field in `QuicStream` and `QuicListener`. No need to do right here and now, I'm making note for myself.